### ISSUE TITLE
No saved changes?

### ISSUE THREAD

**Issue by DevShip** [mbdavid/LiteDB#1431]
I ask for help, I can’t understand why updating records stopped working in V5, in version 4 everything worked fine. In V5, the code is executed. Update signals that changes have been made and re-selection returns the object with the change, but after the connection is destroyed, changes are not made to the database?
Try usage V5 5.0.0 nuget
Try usage V4 4.1.4 nuget

```
void Main()
{
	Run();
	Run();
}

private void Run()
{
	using (var db = new LiteDB.LiteDatabase("FileName=test.db;update=true"))
	{
		var cols = db.GetCollection<User>();
		if (cols.Count() == 0)
			cols.Insert(new User() { Name = "Ivan" });

		var user1 = cols.FindOne(x => x.Name == "Ivan");
		Console.WriteLine($"User1.Age: {user1.Age}");		
		user1.Age++;
		cols.Update(user1);
		var user2 = cols.FindOne(x => x.Name == "Ivan");
		Console.WriteLine($"User2.Age: {user2.Age}");
		Console.WriteLine($"-----");
	}
}

public class User
{
	public LiteDB.ObjectId Id { get; set; }
	public string Name { get; set; }
	public double Age { get; set; }
}
```

V5 Return:
> User1.Age: 0
> User2.Age: 1
> -----
> User1.Age: 0
> User2.Age: 1
> -----

V4 Return:
> User1.Age: 0
> User2.Age: 1
> -----
> User1.Age: 1
> User2.Age: 2
> -----

**Comment by lbnascimento** [comment]
@DevShip Could you test it with v5.0.1?

**Comment by DevShip** [comment]
> @DevShip Could you test it with v5.0.1?

Excellent! Thanks for the responsiveness, the errors were fixed in v5.0.1 and the error was corrected in mode = Share on Netcore 3.1, which led to blocking when .Update () now I was able to return the script with one LiteDabase without reopening the LiteDabase.
